src/main/java/eu/europeana/metis/sandbox/controller/DatasetController.java

```diff
@@ -69,14 +69,6 @@ public DatasetController(HarvestService harvestService,
     this.reportService = reportService;
   }
 
-  /**
```

Comment:
Why are you removing these Javadocs? It's always good to have javadocs to make it easier for others to understand what each method does. Please do not remove them

Reply:
My bad. It was the resolve conflicts. Didn't notice it. I'll fix it